packages/kokoas-server/src/handleRequest/putInvoiceReport/generateInvoicePdf.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getInvoiceById } from 'api-kintone/src/invoice/getInvoiceById';
+import { generateInvoicePdf } from './generateInvoicePdf';
+import { parseInvoiceDat } from './parseInvoiceDat';
+
+describe('file', () => {
+  it('returns an estimate PDF in base64 format', async () => {
+    const recInvoice = await getInvoiceById('5a7a506f-e8b8-42f0-9437-d54c5d790701');
+    const datInvoice = await parseInvoiceDat(recInvoice);
+    const result = await generateInvoicePdf(datInvoice);
+
```

Comment:
stringがtruthyかどうかチェックですね。ありがとうございます。

開発中、ココアスを開かなくても、PDF自体を確認するために、以下のテストはいかがですか。

```suggestion
  const savePath = path.join(__dirname, '../__TEST__', '請求書.pdf');
    const pdfBuffer = Buffer.from(result, 'base64');
    
    // __TEST__　フォルダー に 保存する
    fs.writeFileSync(savePath, pdfBuffer);

    // ファイルが存在するかどうかを確認する
    expect(fs.existsSync('path/to/file')).toBe(true);
```

Reply:
ご指摘ありがとうございます。
上記を参考に[0e1b526](https://github.com/Lorenzras/yumecoco-monorepo/pull/157/commits/0e1b526aa5fa02ac1720fb8541f184fa6622a79b)にて修正しました。